go.mod

```diff
@@ -42,8 +42,10 @@ require (
 	github.com/taskcluster/slugid-go v1.1.0
 	github.com/taskcluster/stateless-dns-go v1.0.6
 	github.com/taskcluster/taskcluster-lib-urls v13.0.0+incompatible
+	github.com/taskcluster/taskcluster-proxy v5.1.0+incompatible
```

Comment:
It looks like something may still be referencing the original taskcluster-proxy repo.